source/src/cfg/cs_Boards.c

```diff
@@ -213,3 +223,77 @@ cs_ret_code_t configure_board(boards_config_t* config) {
 
 	return ERR_SUCCESS;
 }
+
+cs_ret_code_t configure_board_from_uicr(const cs_uicr_data_t* uicrData, boards_config_t* config) {
+	init(config);
+	config->hardwareBoard = uicrData->board;
+
+	switch (uicrData->productRegionFamily.fields.productType) {
+		case PRODUCT_DEV_BOARD:
```

Comment:
There's not one product development board. See e.g. https://github.com/crownstone/bluenet/blob/hardware-support/outlet/source/src/cfg/cs_Boards.c where there's also the asPca10056 with the nRF52840.